script/make-pretty

```diff
@@ -143,10 +144,11 @@ do_shell_check()
     echo -e '     check shell'
     echo -e '====================='
 
-    shfmt -f . | grep -v -E "^($(echo "${OT_EXCLUDE_DIRS[@]}" | tr ' ' '|'))" \
+    local git_files=($(git ls-files))
+    shfmt -f "${git_files[@]}" | grep -v -E "^($(echo "${OT_EXCLUDE_DIRS[@]}" | tr ' ' '|'))" \
```

Comment:
```suggestion
    git ls-files | xargs shfmt -f | grep -v -E "^($(echo "${OT_EXCLUDE_DIRS[@]}" | tr ' ' '|'))" \
```

Reply:
Thought of it too. But not sure if it's a good idea to have 2 `xargs` commands